### PR TITLE
Feature/#26 player computer

### LINKED ISSUE #26: Logic | Player - computer - random answers
old hash: #28

### PR DESCRIPTION
**Pull request by DominikNowak**
closes #26

### CODE REVIEW

File: src/app/logic/ComputerPlayer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import getRandomInt from '../utils/getRandomInt';
+
+export const ComputerPlayer = () => {
+  const player = {
+    askQuestion(question, callback) {
+      if (!question) throw new Error('There is no question');
+      callback(question);
```
Comment (vieraboschkova) [inline src/app/logic/ComputerPlayer.js:7]: What callback could be used for question? What happens after the question is asked?

File: src/app/logic/ComputerPlayer.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    },
+
+    getAnswer(answers, callback) {
+      if (!answers) throw new Error('There is no answers');
+      callback(answers[getRandomInt(0, 3)]);
```
Comment (vieraboschkova) [inline src/app/logic/ComputerPlayer.js:12]: I understand we verify if answer is correct? Or send the answer to be verified? Do we store the number of correct answers or just all answers in computerPlayer object? @Enessetere

**Review by Enessetere — APPROVED** [review]
looks good to me

**Review by SebastianBabinski1 — APPROVED** [review]
In my opinion it's good

**Pull request merged**